databuilder/docs.py

```diff
@@ -31,13 +30,7 @@ def _build_column(name, instance):
 def _build_contracts():
     """Build a dict representation for each Contract"""
 
-    # get all classes from the contracts module
-    contract_classes = inspect.getmembers(contracts, inspect.isclass)
-
-    # make sure we only use subclasses of TableContract
-    contract_classes = [c for _, c in contract_classes if issubclass(c, TableContract)]
-
-    for contract in contract_classes:
+    for contract in TableContract.__subclasses__():
```

Comment:
I'm sure this is the right approach. But do we need to exclude any contracts created as part of testing? I suppose that when this is run "for real" the test modules won't have been loaded. But we might get some weird behaviour during tests.

Reply:
Something to look out for, definitely 👍